lambda_functions/ml_service/ExtractClaim.py

```diff
@@ -2,12 +2,18 @@
 import logging
 import requests
 from bs4 import BeautifulSoup
+from core_layer.handler import item_handler
+from core_layer.db_handler import Session
 
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 
-
 def extract_claim(event, context):
+    with Session() as session:
+        extract_claim(event, context, session)
+
+## Session must be provided from test class
+def extract_claim(event, context, session):
     """extracts claim from item content
```

Comment:
Why using this double structure?
We do not pass _session_ parameter in the other test functions, it should work without.
So the second function signature with _session_ is not needed.
I've tested this in a new branch - it works without error.